{
  try {
    return clazz.newInstance();
  }
 catch (  final Exception ex) {
    throw Util.notExpected(ex);
  }
}
